.github/workflows/publish-toolkit.yml

```diff
@@ -62,7 +62,7 @@ jobs:
         # Attempt to publish the toolkit to PyPI.
         # If the toolkit version already exists, skip publishing.
         # If another error occurs, exit with the error code so that the step fails.
-        PUBLISH_OUTPUT=$(poetry publish --skip-existing 2>&1)
+        PUBLISH_OUTPUT=$(poetry publish --skip-existing 2>&1 || true)
```

Comment:
need to make it a conditional, otherwise when there is an error, the step will just exit without logging the error